packages/material/src/controls/MaterialDateControl.tsx

```diff
@@ -68,6 +68,8 @@ export const MaterialDateControl = (props: ControlProps)=> {
   const format = appliedUiSchemaOptions.dateFormat ?? 'YYYY-MM-DD';
   const saveFormat = appliedUiSchemaOptions.dateSaveFormat ?? 'YYYY-MM-DD';
 
+  const views = appliedUiSchemaOptions.views ?? ['year', 'day'];
```

Comment:
we should probably add month here too, right ;)

Reply:
Actually no. This is the default in @mui/x-date-pickers, too. The initial view is the same for both and looks like this:
![image](https://user-images.githubusercontent.com/6959840/174845833-3886957c-423d-4b44-89d9-cec50966f57e.png)

The only difference is that after clicking on the year and the selecting a year, right now it jumps to the screen shown above (with the changed year ofc). Adding the `'month'` view adds a month selection after selecting the year. IMO this is mostly not needed and just forces the user to select the month again